exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -9,14 +9,16 @@
       <div id="hintsarea"></div>
       <div style="clear: both;"></div>
     </div>
-    <transition id="message" v-show="message" name="expand">{{ message }}</transition>
+    <transition name="expand">
+      <div id="message" v-show="message">{{ message }}</div>
+    </transition>
     <div id="answer-area-wrap">
       <div id="answer-area">
         <div class="info-box">
           <div id="solutionarea"></div>
-          <icon-button @click="checkAnswer" v-show="!complete" class="question-btn" :class="{shaking: shake}" id="check-answer-button">{{ checkText }}</icon-button>
-          <icon-button @click="nextQuestion" v-show="complete" class="question-btn" id="next-question-button">{{ $tr("correct") }}</icon-button>
-          <icon-button v-if="availableHints > 0" @click="takeHint" class="hint-btn">
+          <icon-button @click.native="checkAnswer" v-show="!complete" class="question-btn" :class="{shaking: shake}" id="check-answer-button">{{ checkText }}</icon-button>
```

Comment:
please don't use the `.native` modifier

The `icon-button` emits a `click` event